Doc/using/cmdline.rst

```diff
@@ -539,8 +539,16 @@ Miscellaneous options
      application.  Typical usage is ``python3 -X importtime -c 'import
      asyncio'``.  See also :envvar:`PYTHONPROFILEIMPORTTIME`.
 
+     ``-X importtime=2`` enables additional output that indicates when an
+     imported module has already been loaded.  In such cases, the string
+     ``cached`` will be printed in the self time and cumulative time columns.
+     All other values (``1``, no value, etc.) invoke the original behavior.
```

Comment:
Maybe we should reserve other integer values for future use, and make them show an error message for now?

Reply:
This sounds reasonable. Do you want me to go ahead and implement this or should I want for other opinions?

Reply:
Probably better to wait for other opinions, I won't be able to fully review this PR